root/static/scripts/release-editor/init.js

```diff
@@ -53,12 +53,31 @@ releaseEditor.init = function (options) {
         });
     });
 
+    /*
+     * Allow using range-select (shift-click) on the change recording artist
+     * and change recording title checkboxes in the Recordings page.
+     */
+    utils.withRelease(function (release) {
+        _.each(release.mediums(), function (medium) {
+            _.each(medium.tracks(), function (track) {
+                track.titleDiffersFromRecording();
+                track.artistDiffersFromRecording();
+            });
+        });
+
+        setTimeout(() => {
+            const $checkboxes1 = $('#track-recording-assignation').find('input.update-recording-title[type="checkbox"]');
+            MB.Control.RangeSelect($checkboxes1);
+            const $checkboxes2 = $('#track-recording-assignation').find('input.update-recording-artist[type="checkbox"]');
+            MB.Control.RangeSelect($checkboxes2);
```

Comment:
I'm actually not 100% sure this always returns different DOM nodes when it runs. Or at least I feel uneasy relying on that. If we attach duplicate event handlers on the same checkboxes, that could be bad.

I think it'd be much better to use [event delegation](https://learn.jquery.com/events/event-delegation/) for this: that is, pass the selector string (like `'#track-recording-assignation input.update-recording-title[type="checkbox"]'`)  to `MB.Control.RangeSelect`, then attach a single event handler on the body element that listens for events on that selector. The selector can also be used inside the handler to get all the relevant checkbox nodes.

With that, we wouldn't need this whole `utils.withRelease` hack, since we'd just be attaching an event handler on the body once after the page loads.